front/src/Store.ts

```diff
@@ -20,13 +22,25 @@ const reduxDevToolsOptions: Config = {
   },
 };
 
+const buildListenerMiddleware = () => {
+  const listener = createListenerMiddleware();
+  registerSimulationUpdateInfraIDListeners(listener);
+  registerStdcmUpdateInfraIDListeners(listener);
+
+  return listener;
+};
+
 const middlewares: Middleware[] = [thunk, osrdEditoastApi.middleware];
 
 const store = configureStore({
   reducer: persistedReducer,
   devTools: reduxDevToolsOptions,
   middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware({ serializableCheck: false }).concat(...middlewares),
+    getDefaultMiddleware({ serializableCheck: false })
+      // As mentionned in the Doc(https://redux-toolkit.js.org/api/createListenerMiddleware), Since this can receive actions with functions inside,
```

Comment:
Add a new line and remove capital letter. Nice comment :rocket: 
```suggestion
      // As mentionned in the Doc(https://redux-toolkit.js.org/api/createListenerMiddleware), 
      // since this can receive actions with functions inside,
```